pkg/kv/kvserver/replica_proposal.go

```diff
@@ -655,6 +655,10 @@ func addSSTablePreApply(
 			Size:            sst.BackingFileSize,
 			SmallestUserKey: start.Encode(),
 			LargestUserKey:  end.Encode(),
+
+			// TODO(msbutler): I guess we need to figure out if the backing external
+			// file has point or range keys in the target span.
+			HasPointKey: true,
```

Comment:
It's not possible to have range keys in backup SSTs at the moment right? So this might be sufficient and good enough for us.